jenkins/nightly-21/release-companion.sh

```diff
@@ -14,7 +14,13 @@ source ./version.sh
 
 DESTDIR=/var/www/html/downloads-${version}/nightly/companion
 
+# clean radio source
+cd opentx/radio/src
+make clean
+
```

Comment:
I think it is important to clean the radio dir before the new Companion is built